pkg/buildkit/buildkit.go

```diff
@@ -26,14 +25,11 @@ type Opts struct {
 	KeyPath    string
 }
 
-func InitializeBuildkitConfig(ctx context.Context, c gwclient.Client, image string, manifest *unversioned.UpdateManifest) (*Config, error) {
+func InitializeBuildkitConfig(ctx context.Context, c gwclient.Client, image string) (*Config, error) {
 	// Initialize buildkit config for the target image
 	config := Config{
 		ImageName: image,
-		Platform: ispec.Platform{
-			OS:           "linux",
-			Architecture: manifest.Metadata.Config.Arch,
-		},
+		Platform:  nil,
```

Comment:
do we need to specifically set to nil or can we omit?

Reply:
looks like it actually works without nil, omitted this line